client/src/state/index.js

```diff
@@ -68,8 +105,15 @@ export type MapReference = any
 export type State = {|
   +count: number,
   +notices: {|
-    +list: NoticeMap,
-    +details: NoticeMap,
+    +list: ObjectMap<Notice>,
+    +details: ObjectMap<Notice>,
+  |},
+  +profile: {|
```

Comment:
@Siegrift `{|` means the exact type, so if you have `type A = {| a: any, b: any |}` and you pass `{ a:3, b: 4, c:5 }`, flow will fail because of the `c` prop.